src/app/styles/index.scss

```diff
@@ -27,6 +27,10 @@ li {
   padding: 0;
 }
 
+h2 {
+  margin: 0;
+}
+
```

Comment:
Why is `h1` `margin` reset in `__normalize.scss` but `h2` reset is in the `index.scss` file?